src/plugins/core/merge.ts

```diff
@@ -257,8 +257,12 @@ export class MergePlugin extends CorePlugin<MergeState> implements MergeState {
     if (!this.isInMerge(position)) {
       return position;
     }
-    const mergeTopLeftPos = this.getMerge(position)!.topLeft;
-    return { sheetId: position.sheetId, col: mergeTopLeftPos.col, row: mergeTopLeftPos.row };
+    const mergeZone = this.getMerge(position);
+    return {
+      sheetId: position.sheetId,
+      col: mergeZone?.left as HeaderIndex,
+      row: mergeZone?.top as HeaderIndex,
```

Comment:
As a general rule we should avoid casting unless there is no other choice.

An `if(mergeZone)` just above instead of the `if(this.isInMerge)` would fix that.

Note that this wasn't needed because we had an `!` before, which is something we should also avoid